Thing.js

```diff
@@ -0,0 +1,5 @@
+const btn =document.querySelector('button');
```

Comment:
Little formatting for you

```suggestion
const btn = document.querySelector('button');
```